distributed/src/main/scala/asobu/distributed/EndpointRegistry.scala

```diff
@@ -0,0 +1,27 @@
+package asobu.distributed
+
+import javax.inject.{Inject, Provider, Singleton}
+
+import akka.actor._
+import akka.cluster.Cluster
+
+import scala.concurrent.{Await, Promise}
+import scala.concurrent.duration._
+
+class EndpointRegistry extends Actor with ActorLogging {
+  def receive: Receive = ???
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Usage of ??? operator](https://www.codacy.com/app/kailuo-wang/asobu/file/1695710743/issues/source?bid=3037259&fileBranchId=3044589#l12)